src/javascript/app/base/client.js

```diff
@@ -86,6 +88,33 @@ const Client = (() => {
         }
     };
 
+    // Called when logging out to end ongoing chats if there is any
+    const endLiveChat = () => new Promise ((resolve) => {
```

Comment:
why did we decide to move this here?

Reply:
Cause importing from livechat.js into client.js causes a circular dependency...client -> livechat -> traffic source ->client. So I moved this here to break that chain